packages/hub/tasks/send-email-card-drop-verification.ts

```diff
@@ -3,6 +3,7 @@ import config from 'config';
 import { query } from '@cardstack/hub/queries';
 import { inject } from '@cardstack/di';
 import * as Sentry from '@sentry/node';
+import { formatDuration } from 'date-fns'
```

Comment:
Do we need an external library? I tried out `formatDuration`. It doesn't provide anything beyond `minute` -> `minutes`.

Reply:
it was already present but really I’m just missing Rails-y niceties that would turn 90 into `1 hour and 30 minutes` or whatever… but we can think about that in the future if the expiry window changes.